package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "bench": "node bench",
-    "test": "node test"
+    "test": "node test/test.js"
```

Comment:
Can you please put the test back? We don't have more files for testing to justify folder for now, and this makes Github unhappy with diffs.